src/test/java/org/nikita/chiken_bell/core/entity/CartTest.java

```diff
@@ -0,0 +1,40 @@
+package org.nikita.chiken_bell.core.entity;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CartTest {
+
+    private static Cart cart;
```

Comment:
Why is it static?

Reply:
because annotation @BeforeAll should be static method and this method use Cart item